### DISCUSSION TITLE
OmniWall

### DISCUSSION THREAD

**Discussion by troy-jacobson** [aderusha/DDD-Printable-Wall-Control-System#33] [Q&A]
Hi,
This looks like a cool project and something I could really use.  However, I have some Omniwall, which is similar to the panels used here, but the geometry of the slots is slightly different.

Is there any way to do a bulk edit in Sketchup of all of the tabs?  They need to be about 10mm shorter, I think.

Their position may also need to be tweaked up or down so the locking pieces align with the holes, but that should be simple once I can do the first part.

**Comment by mathrock** [comment]
I also have an OmniWall and recently came across these models which are great! I think with some minor tweaks to the quickhooks and side pieces you could adapt everything to work with the OmniWall.  

Is there some measurements I could help provide? Or take detailed pictures of anything?

Here's the board itself, seems to be 1" spacing, with 1/8" width slots, and standard 1/4" peg holes.

![IMG_20220319_170557](https://user-images.githubusercontent.com/30048/159139337-1728d70d-a311-42b7-a847-1af311e6ee52.jpg)
![IMG_20220319_170641](https://user-images.githubusercontent.com/30048/159139342-34422703-7c62-4373-96b2-ab61d57554f4.jpg)
![IMG_20220319_170717](https://user-images.githubusercontent.com/30048/159139346-69cdd26a-af55-47eb-ad14-485cf7a70f97.jpg)

The OmniWall hooks look like this:
![IMG_20220319_174456__01](https://user-images.githubusercontent.com/30048/159139453-4db20c35-7e02-4399-8cab-cbb96ab5b675.jpg)
![IMG_20220319_174440__01](https://user-images.githubusercontent.com/30048/159139457-57df7cfd-7e4f-4a53-8ea5-2aa204749f42.jpg)
